src/editor/Editor.js

```diff
@@ -435,6 +433,17 @@ define(function (require, exports, module) {
 
         this.on("cursorActivity", function (event, editor) {
             self._handleCursorActivity(event);
+
+            var providersFound = MainViewManager._findProvider(editor);
+            if (providersFound.length) {
+                self.trigger("providerAvailable", self, providersFound);
+
+                var i,
+                    len;
+                for (i = 0, len = providersFound.length ; i < len ; i++) {
+                    console.log(providersFound[i].name+" ProviderFound");
```

Comment:
NOTE: with `console.log()` you can just pass it any number of arguments separated by `,`: `console.log(one, two, three)` and it will concat for you.